app/uk/gov/hmrc/plasticpackagingtax/returns/views/components/returnsSectionHeading.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.plasticpackagingtax.returns.models.obligations.Obligation
+@import uk.gov.hmrc.plasticpackagingtax.returns.views.html.components.sectionHeader
+
+@this(
+        sectionHeader: sectionHeader,
+)
+
+@(obligation: Obligation)(implicit messages: Messages)
+
+@sectionHeader(message =
+    messages("returns.sectionHeader",
```

Comment:
is this key already in messages?

Reply:
Nope. I was copying bits across from my feature branch and missed it. Added now.